ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```
@@ -615,6 +615,16 @@ objects:
 
           echo "Deprovisioning cluster ..."
           openshift-install --dir /tmp/artifacts/installer --log-level=debug destroy cluster
+
+          declare -ra to_be_deleted=(
+            "${TMP_ARTIFACTS}/installer/auth"
```

`${TMP_ARTIFACTS}` is undefined here.

```suggestion
            "/tmp/artifacts/installer/auth"
```

---

woops. Thank you!